running-code/execute-package.md

```diff
@@ -0,0 +1,163 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(execute-package)=
+# How to Execute a Python Package
+
+In [How to Execute a Python Script](execute-script) you learned about two primary ways to execute a stand-alone Python script.
+There are two other ways to execute Python code as commands, both of which work for code that has been formatted as a package.
+
+1. You can execute modules using their import name
+2. A package can provide arbitraty command names that execute parts of themselves
+
+## 1. Executable modules
+
+In the [Pass your script to the Python command lesson](execute-script-pass-to-python) you learned that the `python` command can
+be passed a file path for execution. Alternatively you can also pass the name of a module, exactly as would be used after an `import`.
+In this case, Python will look up the module referenced in the
+[currently active environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#create-and-use-virtual-environments),
+and will execute it as a script.
+
+This execution mode is performed with the `-m` flag, as in `python -m site`. It can be used in place of a file
+path, but cannot be used in combination with a path, as there can only be one executing module.
+
```

Comment:
Reading down further, i see that this was explained in executing scripts lesson, my bad - imo it fits a bit better in this one, since typically one doesn't import scripts